src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -91,11 +91,11 @@ public class InventoryBrowser extends ModalJFrame implements InventoryListener {
 			MessageBundle.getMessage("angal.inventory.status.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.common.user.col").toUpperCase() };
 	private int[] pColumwidth = { 150, 150, 150, 200 };
-	private JComboBox stateComboBox;
+	private JComboBox<String> stateComboBox;
 	private JLabel stateLabel;
 	JButton next;
 	JButton previous;
-	JComboBox pagesCombo = new JComboBox();
+	JComboBox<Integer> pagesCombo = new JComboBox<Integer>();
```

Comment:
Why the use of objects when primitives would work????
```suggestion
	JComboBox<int> pagesCombo = new JComboBox<Integer>();
```